sdk/storage/storage-blob/karma.conf.js

```diff
@@ -36,7 +36,7 @@ module.exports = function(config) {
     files: [
       // polyfill service supporting IE11 missing features
       // Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys,Symbol.iterator
-      "https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys,Symbol.iterator|always",
+      "https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys|always,Symbol.iterator",
```

Comment:
From blob - 12.1
 - `https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys|always`


In this PR
- Old    -  `https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys,Symbol.iterator|always`
- New - `https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.assign,Object.keys|always,Symbol.iterator`


If you compare the content from these links, `From blob - 12.1` and `New - In this PR` are exactly same.

Reply:
In case the polyfill is being changed, 
- please make sure the tests succeed in IE
- and update this in the readmes - https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob#compatible-with-ie11

Reply:
- [ ] Why is this required?
Tests on `PagedAsyncIterableIterator` throw `TypeError: Symbol.iterator is not defined`.
We have #3893 merged, but I haven't totally figure it out.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/8168282/76361618-59061a00-635a-11ea-90f6-e8f555162ead.png)

- [ ] please make sure the tests succeed in IE

- [ ] update this in the readme

Reply:
Thanks @ljian3377, this makes sense now. 

This test has been there for a long period, it hadn't failed previously to my recollection.
Would like to know the root cause.. probably something might've changed in `core-paging` or `core-asynciterator-polyfill`. Are you aware of any changes, @jeremymeng ?

Reply:
There are changes in async iterators from new version of typescript. We also made changes to code-paging.  I will log an issue but priority to investigate would be low since it's IE 11 only I belive.

Reply:
Logged #7752